TileFormats/VectorData/README.md

```diff
@@ -1,19 +1,248 @@
+| `batchTableJSONByteLength` | `uint32` | The length of the batch table JSON section in bytes. Zero indicates that there is no batch table. |
+| `batchTableBinaryByteLength` | `uint32` | The length of the batch table binary section in bytes. If `batchTableJSONByteLength` is zero, this will also be zero. | 
+| `polygonIndicesByteLength` | `uint32` | The length of the polygon indices buffer. |
+| `polygonPositionsByteLength` | `uint32` | The length of the polygon positions buffer. |
+| `polylinePositionsByteLength` | `uint32` | The length of the polyline positions buffer. |
+| `pointPositionsByteLength` | `uint32` | The length of the point positions buffer. |
+
+If `featureTableJSONByteLength` equals zero, the tile does not need to be rendered.
+
+The body section immediately follows the header section, and is composed, in order, of four fields: `Feature Table`, `Batch Table`, `Indices`, `Positions`, and `Meshes`.
+
+Code for reading the header can be found in
+[Vector3DModelTileContent.js](https://github.com/AnalyticalGraphicsInc/cesium/blob/vector-tiles/Source/Scene/Vector3DTileContent.js)
+in the Cesium implementation of 3D Tiles.
+
+## Feature Table
+
+Contains values for `vctr` semantics used to render features.  The general layout of a Feature Table is described in the [Feature Table specification](../FeatureTable).
+
+The `vctr` Feature Table JSON schema is defined in [vctr.featureTable.schema.json](../../schema/vctr.featureTable.schema.json).
+
+### Semantics
+
+Per-feature semantics specific to a feature type are prefixed with the name of the feature type. e.g. `POLYGON` for pre-triangulated polygons, `POLYLINE` for polylines and `POINT` for points.
+
+At least one global `LENGTH` semantic must be defined. 
+If `POLYGONS_LENGTH` is not defined, or zero, no polygons will be rendered. 
+If `POLYLINES_LENGTH` is not defined, or zero, no polylines will be rendered.
+If `POINTS_LENGTH` is not defined, or zero, no points will be rendered.
+If `MESHES_LENGTH` in not defined, or zero, no meshes will be rendered.
+If `BOXES_LENGTH` in not defined, or zero, no boxes will be rendered.
+If `CYLINDERS_LENGTH` in not defined, or zero, no cylinders will be rendered.
+If `ELLIPSOIDS_LENGTH` in not defined, or zero, no ellipsoids will be rendered.
+If `SPHERES_LENGTH` in not defined, or zero, no spheres will be rendered.
+Multiple feature types may be defined in a single Vector tile using multiple `LENGTH` semantics, and in that case, all specified feature types will be rendered.
+
+If a semantic has a dependency on another semantic, that semantic must be defined as well.
+
+#### Global Semantics
+
+The semantics define global properties for all vector elements.
+
+| Semantic | Data Type | Description | Required |
+| --- | --- | --- | --- |
+| `POLYGONS_LENGTH` | `uint32` | The number of pre-triangulated polygons in the tile. | :red_circle: No. |
```

Comment:
Here and throughout I think the cleanest thing to do is to not have the period for simple "Yes" and "No" and only have it when it is a full sentence.